internal/namespaces/instance/v1/custom_server_test.go

```diff
@@ -27,7 +27,8 @@ func Test_ServerVolumeUpdate(t *testing.T) {
 				assert.Equal(t, 10*scw.GB, ctx.Result.(*instance.AttachVolumeResponse).Server.Volumes["1"].Size)
 				assert.Equal(t, instance.VolumeTypeBSSD, ctx.Result.(*instance.AttachVolumeResponse).Server.Volumes["1"].VolumeType)
 			},
-			AfterFunc: deleteServer("Server"),
+			AfterFunc:       deleteServer("Server"),
+			DisableParallel: true,
```

Comment:
Why do you need to disable it?

Reply:
It seems the tests is flaky, works better by disabling parallel